lib/cancel/Cancel.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+/**
+ * A `Cancel` is an object that is thrown when an operation is canceled.
+ *
+ * @class
+ * @param {string=} message The message.
+ */
+function Cancel(message) {
```

Comment:
`Cancel` must not derive from `Error` according the cancelable promises spec. Please see [this](https://github.com/tc39/proposal-cancelable-promises/blob/master/Not%20an%20Error.md#api-design) for details.